src/scss/_placeholders.scss

```diff
@@ -32,6 +32,12 @@ $_o-labels-first-import: true !default;
 		left: 0;
 		width: calc(#{$_o-labels-indicator-circle-size-calc});
 		height: calc(#{$_o-labels-indicator-circle-size-calc});
+		// At some font sizes the width/height of the circle is a decimal.
+		// For small font sizes, i.e 14px, the browser renders the circle
+		// squashed, even though the width/height are equal. Setting the
+		// transform property changes the way the browser renders the circle,
+		// in a way which overcomes this problem
+		transform: scale(1);
```

Comment:
can we have a little more information about what "changes the way the browser renders the circle" means? perhaps a link to the relevant part of the specification or browser docs?

does setting `opacity: 1` also work?

Reply:
Honestly I'm not sure. I think it might be to do with offloading to the GPU. Setting `opacity: 1` does nothing but setting `will-change: transform` has the same effect

Reply:
I guess a less hacky solution would be to make the indicator labels less flexible and have different classes which set whole values. `o-labels-indicator--xs`, `o-labels-indicator--s`, `o-labels-indicator--m`.

Reply:
okay, so in Blink we're pushing it onto a compositing layer so that it's handled by the GPU and does hardware accelerated subpixel rendering.
i think that `will-change: opacity` would also make it work then, but the behaviours are documented in the rendering engines rather than specifications.
mostly this is browser internals stuff, but there is a little written about it in the [spec for will-change](https://drafts.csswg.org/css-will-change/#will-change).
i do prefer the idea of having classes that set whole values, but i'm going to hit approve on this pr.

Reply:
Looks like this still doesn't work in Safari/Webkit. Change of plan, let's avoid whole numbers: https://github.com/Financial-Times/o-labels/pull/104